tss-esapi/src/context/tpm_commands/non_volatile_storage.rs

```diff
@@ -97,6 +233,86 @@ impl Context {
     /// # Details
     /// This method is used to read the public
     /// area and name of a nv index.
+    ///
+    /// # Arguments
+    /// * `nv_index_handle` - The [NvIndexHandle] associated with NV memory
+    ///                       for which the public part is to be read.
+    /// # Returns
+    /// A tuple containing the public area and the name of an nv index.
+    ///
+    /// # Example
+    /// ```rust
+    /// # use tss_esapi::{
+    /// #     Context, TctiNameConf, attributes::{SessionAttributes, NvIndexAttributes},
+    /// #     handles::NvIndexTpmHandle, interface_types::algorithm::HashingAlgorithm,
+    /// #     structures::{SymmetricDefinition, NvPublic}, constants::SessionType,
+    /// # };
+    /// use tss_esapi::{
+    ///       interface_types::resource_handles::Provision,
+    /// };
```

Comment:
Did you mean to display here all the required imports for the visible code below?

Reply:
Yes, I wanted to show what is needed for the code that is visible.